docs/EN_US/RoxieReference/RoxieRefMods/RoxieCopySettings.xml

```diff
@@ -111,7 +111,7 @@
     not otherwise specified Roxie copies from clusters in its own environment
     first. To enable the Preferred Clusters feature, you can use the HPCC
     Systems Configuration Manager. (for additional information about
-    Configuring your HPCC System see <emphasis>Installing and Running the HPCC
+    Configuring your HPCC Systems see <emphasis>Installing and Running the HPCC Systems
```

Comment:
This no longer reads correctly.  your HPCC Systems system?  your HPCC Systems platform.

Reply:
or remove "your"